ui/src/app/(index)/_components/AfterLogin.tsx

```diff
@@ -0,0 +1,127 @@
+import User from '@/access/User';
+import Role from '@/access/Role';
+import Image from 'next/image';
+import { KeyRound } from 'lucide-react';
+import Link from 'next/link';
+import { Button } from '@/components/ui/button';
+
+const AfterLogin = ({
+    currentUser,
+    numberOfGroupings,
+    numberOfMemberships
+}: {
+    currentUser: User,
+    numberOfGroupings: number,
+    numberOfMemberships: number
+}) => {
+    const getHighestRole = () => {
+        if (currentUser.roles.includes(Role.ADMIN)) return 'Admin';
+        else if (currentUser.roles.includes(Role.OWNER)) return 'Owner';
+        else return 'Member';
+    }
+
+    const items = [
```

Comment:
Let's call this variable `pageInfoItems` to be more specific. Then below when you use the `.map` function, the `item` will be called `pageInfoItem`.